README.md

```diff
@@ -39,6 +39,10 @@ To get a local copy up and running follow these simple example steps.
 👤 Hamza Bin Ali Tarar
 
 - GitHub: [@Lockless404](https://github.com/lockless404)
```

Comment:
- [ ] Please add  to your readme file how you can download the project and run it locally.